### DISCUSSION TITLE
Some properties of devcontainer.json could not be validated.

### DISCUSSION THREAD

**Discussion by 5885885** [community/community#145863] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi
When I logged in today I was prompted to install an update. After installing it I got an error message: "2024-11-29 14:02:02.012Z: Some properties of devcontainer.json could not be validated". 

The codespaces can no longer be accessed by the CLI i think. Or it could just be that I have gone astray and cant find it again. How can i access it again using the CLI?

**Comment by 5885885** [comment]
When i refresh the page i get the message: "This codespace is currently running in recovery mode due to a configuration error. Please review the creation logs, update your dev container configuration as needed, and run the "Rebuild Container" command to rectify."

When I check github.com/codespaces/ i can see:

"Owned by code50
code50
https://github.com/code50/178682147
[effective space yodel](https://github.com/codespaces/effective-space-yodel-5gp6764645vfxw6)
main
*

This codespace has uncommitted changes"

Not sure what to do here.

**Reply by 5885885** [reply]
I found it. I could reopen through some menu. Still gets error messages but at least i found the way back to the codespaces.

**Reply by 5885885** [reply, marked as answer]
I found the fix. 

When in codespaces press ctrl+shift+P and search Remote-Containers: Rebuild Container. Run it.

**Comment by listen-asko** [comment]
Your GitHub Codespace is currently in recovery mode due to a configuration error. To resolve this:
  1. View Creation Logs: Check the creation logs on GitHub Codespaces to identify the configuration issue.
  2. Update Configuration: Based on the logs, update your configuration files (e.g.,devcontainer.json) as needed.
  3. Handle Uncommitted Changes: Decide whether to commit or discard any uncommitted changes in your codespace.
  4. Rebuild Container: Once the configuration is updated, run the "Rebuild Container" command from the GitHub Codespaces interface.
If the issue persists, contact GitHub Support for further assistance.

**Reply by mavli1999** [reply]
but  how to update devcontainer.json? Update it to what exactly?

**Comment by tashveekhare** [comment]
Was anyone able to find a resolution to this?

**Reply by 5885885** [reply]
I found the fix. 

When in codespaces press ctrl+shift+P and search Remote-Containers: Rebuild Container. Run it.
